packages/cli/templates/form-trigger.handlebars

```diff
@@ -416,6 +528,28 @@
 				}
 			}
 
+			function validateEmailInput(input, errorElement) {
+				const value = input.value.trim();
+
+				if (value !== '') {
+					const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
this regex is very simple and does allow invalid email addresses, for example ".@..."
```suggestion
					const regex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```